### ISSUE TITLE
Could you provide source code downloads?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [piamay/squeezelite#23]
```
Hello,

I maintain the Gentoo ebuild for this 
(http://git.overlays.gentoo.org/gitweb/?p=user/squeezebox.git;a=summary). 
Gentoo, of course, wants to build from source so having an archive with the 
source is necessary.

I've previously been downloading an archive generated on-the-fly from a git 
commit that I hope corresponds to the version you've released binaries of, but 
this has exposed a problem.

Google code seems to generate the archives on the fly and they seem to be 
subtly different sometimes even when generated from the same commit. This 
causes the Gentoo ebuild to fail because I have to store a hash of the expected 
archive contents to ensure the archive is downloaded correctly.

So, in summary, could you commit a tarball of the source corresponding to the 
current version to the 'downloads' section in the same way as you do for the 
binary releases, please?

Thanks in advance if you're able to do this.
```

Original issue reported on code.google.com by `DrSpot...@gmail.com` on 21 Apr 2013 at 6:50
